### PR TITLE
Web feature: 1D jump to sequence and splash page

### PR DESCRIPTION
**Pull request by FontenotZach**
Description:  
- New functionality allowing for jumping to sequence on 1D graph has been added and tested with bounds testing.  
- GUI changes were made to both the graph view and the home page.  
- The graph view sidebar now contains two scrollable elements for BED files and controls.  
- The home page now contains simple interface for file input and a sidebar for links.
- Added bootstrap dependency.
- Beautified index.html with VSCode plugin.

Type of changes: 
- Feature
- GUI

Testing: 
- GFA, TSV, and BED file upload produce expected behavior.
- Bounds testing on 1D graph demonstrates no input will fall outside the 1D graph bounds.

Additional notes: 
- New typescript code is commented.
- No new compiler/browser warnings or errors.
- The logo next to 'WaraGraph' on the home page is AI generated and copyright free.  

Concerns:
- Not sure about the color scheme or university affiliation.  I went with UTHSC for now.  
- Not sure how we are versioning this software, so I just wrote 'waragraph web beta' at the bottom.
- Not sure why package-lock.json changed so much for one dependency update. Is this ok?

Screenshots: 
- Here is the home page.
image
- Here is the graph page, note that the former input boxes are now read-only.
image

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -1,58 +1,143 @@
...
+
+        <hr class="mb-0">
+        <div>
+          <span>waragraph web beta</span>
+        </div>
       </div>
+      <div class="d-flex flex-column" style="flex-grow: 1; height: 100vh;">
+        <div id="home-content-top" class="flex-column" style="flex-grow: 1; overflow-y: auto;">
+          <div class="h2 mt-5 mb-1 home-content-top-item">
+            <img src="icons\logo-2.png" style="margin: 2px;" alt="icon" height="60" width="60">
```
Comment (chfi) [inline web/index.html]: Path should be `icons/logo-2.png` for portability

File: web/index.html [hunk 2]
```diff
@@ -1,58 +1,143 @@
...
+            <span class="h1 mb-0" style="color: black; font-size: 60px;">WaraGraph</span>
```
Comment (chfi) [inline web/index.html]: "Waragraph" for consistency (and aesthetics imo)

**Review by chfi — CHANGES_REQUESTED** [review]
Great! Besides the two nitpicks below, another small issue is that the rows in the BED list have the same color when toggled on and off.

Also, this can be a separate PR (and isn't high priority), but it'd be nice if the BED and control panels were resizable.

**Comment by FontenotZach** [comment]
Thanks for the review! HTML changes are in, but I wasn't sure what you meant about the BED list color and the toggle. Is that all in the BED pane or somewhere else?

**Review by chfi — APPROVED** [review]

**Comment by chfi** [comment]
> Thanks for the review! HTML changes are in, but I wasn't sure what you meant about the BED list color and the toggle. Is that all in the BED pane or somewhere else?

When clicking on the entries in the BED list, the corresponding annotation visualization is toggled on/off: https://github.com/chfi/waragraph/blob/web/web/ts/sidebar-bed.ts#L376-L408
The CSS is in sidebar-bed.css https://github.com/chfi/waragraph/blob/web/web/sidebar-bed.css#L42-L45

**Pull request merged**